src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -67,6 +67,10 @@ public function denormalize($data, $class, $format = null, array $context = arra
     {
         $dateTimeFormat = isset($context[self::FORMAT_KEY]) ? $context[self::FORMAT_KEY] : null;
 
+        if (empty($data)) {
```

Comment:
Could be too broad? Should should only `null`? Should we also really check the empty string?
@dunglas any hint?

Reply:
I agree. Maybe should we just tweak the `supportDenormalization` function to skip this normalizer is the value isn't a string.

Reply:
And we keep the test on empty strings or do we add it in the `supportDenormalization` ?

Reply:
isn't that a BC ?

Reply:
done, added to supportDenorm and modifed the denormalize